less/_grid.less

```diff
@@ -188,20 +188,31 @@
 	}
 	// when the page has a max-width set
 	&.max-width {
-		@max-width: 1280px;
```

Comment:
removed this because this lives in the skin file. also having it here defeats the purpose of the value in the skin.less file